src/navigate-pop/using-workspaces.md

```diff
@@ -0,0 +1,48 @@
+# Workspaces
+
+Workspaces serve to reduce window clutter, especially on smaller displays. 
+
+---
+
+## Using Workspaces
+
+Show and access workspaces by clicking on the workspaces icon in the dock, clicking `Workspaces` in the top left corner of your screen, or by pressing `SUPER` + `D`.
+
+The open windows for a given workspace will display when that workspace is selected.
+
+<video autoplay loop>
+        <source src="/images/workspaces/show-workspaces.webm" />
+</video>
+
+Click and drag applications between workspaces as needed.
+
+<video autoplay loop>
+        <source src="/images/workspaces/drag-window.webm" />
+</video>
+
+Selecting a window and pressing `SUPER` + `Shift` + the `🠑` or `🠓` keys will move a window to a different workspace.
+
+<video autoplay loop>
+        <source src="/images/workspaces/move-new-workspace.webm" />
+</video>
+
+Press the `SUPER` + `Ctrl` + the `🠑` or `🠓` to move between workspaces.
+
+<video autoplay loop>
+        <source src="/images/workspaces/switch-workspaces.webm" />
```

Comment:
Do you know why the animations aren't showing up? That's a problem.

Also I just noticed, you're using the dark theme, but you have the light theme background showing. (Looks like you're on 21.10, which we no longer support, based on what's in the Files sidebar.) I would highly recommend making videos in 22.04 LTS since it is the current version and will have at least a few years of support.

Reply:
This mdbook started before 22.04 was released. I thought I should be consistent instead of switching halfway. I can use 22.04 going forward, but we should create an issue on the repo to retroactively check and update prior sections.

Reply:
I'll redo all these webms